source/connect/connection-options/connection-pools.txt

```diff
@@ -165,6 +165,18 @@ object:
      minPoolSize: 10
    });
 
+.. note::
+
+   A ``MongoClient`` configured with ``maxIdleTimeMS`` and ``minPoolSize`` 
+   set to ``0`` is optimal for workloads with sustained periods of low activity. 
+   This configuration allows the connection pool to close unused connections 
+   during periods of inactivity. 
+   
+   In versions earlier than 6.18.0, the {+driver-short+} did not close idle
+   connections when ``minPoolSize`` was set to ``0``. Starting in version
+   6.18.0, the connection pool correctly closes idle connections regardless
+   of the ``minPoolSize`` setting.
```

Comment:
```suggestion
   In versions earlier than 6.18.0, the {+driver-short+} did not close idle
   connections when ``minPoolSize`` was set to ``0`` during periods of inactivity. Starting in version
   6.18.0, the connection pool correctly closes idle connections regardless
   of the ``minPoolSize`` setting.
```

just a slight clarification: idle connections were pruned _if_ there was activity, its only when there was no activity or minimal activity that the connections were not closed.